sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -1184,3 +1201,15 @@ func withUnmaskedPaths(unmaskPaths []string) oci.SpecOpts {
 		return nil
 	}
 }
+
+// Convert labels to map[string]string for containerd.WithPullLabels.
+// Labels are in the format of "key=value,key2=value2".
+func convertLabels(labels string) map[string]string {
+	labelsMap := make(map[string]string)
+	labelsList := strings.Split(labels, ",")
+	for _, label := range labelsList {
+		labelKeyValue := strings.Split(label, "=")
+		labelsMap[labelKeyValue[0]] = labelKeyValue[1]
```

Comment:
I believe it is here: https://github.com/opencontainers/image-spec/blob/main/config.md#:~:text=creating%20a%20container.-,Labels,-object%2C%20OPTIONAL

which links to https://github.com/opencontainers/image-spec/blob/365fa41472a19107b0b4cbc17e141a9b97f9b66e/annotations.md#rules

Reply:
containerd expects a map[string]string as labels input. Otherwise, all it cares about is that the key and value are under 4096 bytes combined. https://github.com/containerd/containerd/blob/f5e7fe0cb6286333a597a407029fe7c760626669/pkg/labels/validate.go#L32-L41

I'll add this validation and meanwhile I'll validate `key=value` format.